csharp/extractor/Semmle.Extraction.CSharp/Tuples.cs

```diff
@@ -32,8 +32,8 @@ internal static class Tuples
         internal static void array_element_type(this TextWriter trapFile, ArrayType array, int dimension, int rank, Type elementType) =>
             trapFile.WriteTuple("array_element_type", array, dimension, rank, elementType);
 
-        internal static void attributes(this TextWriter trapFile, Attribute attribute, Type attributeType, IEntity entity) =>
-            trapFile.WriteTuple("attributes", attribute, attributeType, entity);
+        internal static void attributes(this TextWriter trapFile, Attribute attribute, AttributeKind kind, Type attributeType, IEntity entity) =>
+            trapFile.WriteTuple("attributes", attribute, (int)kind, attributeType, entity);
```

Comment:
The `int` cast is not strictly necessary.

Reply:
Ah yes - it looks like the WriteColumn extensions method does the cast for us.
Will fix.